test/RadworksGovernor.t.sol

```diff
@@ -137,10 +138,589 @@ abstract contract Propose is ProposalTest {
+    bytes[] memory _calldatas = new bytes[](3);
+    string memory _description = "Update governance settings";
+
+    _targets[0] = address(governorBravo);
+    _calldatas[0] = _buildProposalData("setVotingDelay(uint256)", abi.encode(_newDelay));
+
+    _targets[1] = address(governorBravo);
+    _calldatas[1] = _buildProposalData("setVotingPeriod(uint256)", abi.encode(_newVotingPeriod));
+
+    _targets[2] = address(governorBravo);
+    _calldatas[2] =
+      _buildProposalData("setProposalThreshold(uint256)", abi.encode(_newProposalThreshold));
+
+    // Submit the new proposal
+    vm.prank(PROPOSER);
+    uint256 _newProposalId = governorBravo.propose(_targets, _values, _calldatas, _description);
+
+    // Ensure proposal is in the expected state
+    IGovernor.ProposalState _state = governorBravo.state(_newProposalId);
+    assertEq(_state, IGovernor.ProposalState.Pending);
+
+    _jumpToActiveProposal(_newProposalId);
+
+    _delegatesCastVoteOnBravoGovernor(_newProposalId, FOR);
+    _jumpToVotingComplete(_newProposalId);
+
+    // Ensure the proposal has succeeded
+    _state = governorBravo.state(_newProposalId);
+    assertEq(_state, IGovernor.ProposalState.Succeeded);
+
+    // Queue the proposal
+    governorBravo.queue(_targets, _values, _calldatas, keccak256(bytes(_description)));
+
+    // Ensure the proposal is queued
+    _state = governorBravo.state(_newProposalId);
+    assertEq(_state, IGovernor.ProposalState.Queued);
+
+    _jumpPastProposalEta(_newProposalId);
+
+    // Execute the proposal
+    governorBravo.execute(_targets, _values, _calldatas, keccak256(bytes(_description)));
+
+    // Ensure the proposal is executed
+    _state = governorBravo.state(_newProposalId);
+    assertEq(_state, IGovernor.ProposalState.Executed);
+
+    // Confirm that governance settings have updated.
+    assertEq(governorBravo.votingDelay(), _newDelay);
+    assertEq(governorBravo.votingPeriod(), _newVotingPeriod);
+    assertEq(governorBravo.proposalThreshold(), _newProposalThreshold);
+  }
+
+  function testFuzz_NewGovernorCanPassMixedProposal(
+    uint256 _amount,
+    address _receiver,
+    uint256 _seed
+  ) public {
+    IERC20 _token = _randomERC20Token(_seed);
+    _assumeReceiver(_receiver);
+    uint256 _timelockTokenBalance = _token.balanceOf(TIMELOCK);
+
+    // bound by the number of tokens the timelock currently controls
+    _amount = bound(_amount, 0, _timelockTokenBalance);
+    uint256 _initialTokenBalance = _token.balanceOf(_receiver);
+
+    _upgradeToBravoGovernor();
+    (
+      uint256 _newProposalId,
+      address[] memory _targets,
+      uint256[] memory _values,
+      bytes[] memory _calldatas,
+      string memory _description
+    ) = _submitTokenSendProposalToGovernorBravo(address(_token), _amount, _receiver);
+
+    _jumpToActiveProposal(_newProposalId);
+
+    // Delegates vote with a mix of For/Against/Abstain with For winning.
+    // TODO: Need more delegates for this.. pool together had more!
```

Comment:
just curious, can you elaborate on why we need more delegates?

Reply:
We should be able to pull all of the "need more delegates" in a future PR, as we've been able to test with the delegates we have.

Reply:
i'm confused why we are adding these comments if we don't need them! i won't block merge on it though.